frontend/src/app/components/channel/channel.html

```diff
@@ -1,35 +1,58 @@
-<section class="md-whiteframe-z1" style="margin-right: 4px;">
-  <md-content flex>
-    <md-grid-list
-      md-cols="1"
-      md-row-height="16:9"
-      md-gutter="3px">
-      <md-grid-tile
-        ng-style="{
-        'background': 'red'
-        }"
-        md-colspan="1"
-        md-rowspan="1">
-
-        <video class="program" style="width: 100%" autoplay></video>
-      </md-grid-tile>
-      <md-grid-tile
-        ng-style="{
-        'background': 'green'
-        }"
-        md-colspan="1"
-        md-rowspan="1">
-
-        <video class="preview" style="width: 100%" autoplay></video>
-      </md-grid-tile>
-    </md-grid-list>
-
-    <md-list>
-      <md-list-item ng-repeat="user in vm.users">
-        <div class="md-list-item-text" layout="column">
+<section class="channel md-whiteframe-z1">
+  <md-grid-list md-cols="1" md-row-height="16:9" md-gutter="3px">
+    <md-grid-tile
+      ng-style="{
+      'background': 'red'
+      }"
+      md-colspan="1"
+      md-rowspan="1">
+
+      <video class="program" style="width: 100%" poster="data:image/gif,AAAA" autoplay></video>
+
+      <md-grid-tile-header>
+        <h3>PROGRAM {{vm.programUser.title}}</h3>
+      </md-grid-tile-header>
+    </md-grid-tile>
+
+    <md-grid-tile
+      ng-style="{
+      'background': 'green'
+      }"
+      md-colspan="1"
+      md-rowspan="1"
+      ng-click="vm.next()">
+
+      <video class="preview" style="width: 100%" poster="data:image/gif,AAAA" autoplay></video>
+
+      <md-grid-tile-header>
+        <h3>PREVIEW {{vm.previewUser.title}}</h3>
+      </md-grid-tile-header>
+    </md-grid-tile>
+  </md-grid-list>
+
+  <md-list class="users">
+    <!-- Online users -->
+    <md-list-item ng-repeat="user in vm.onlineUsers">
+      <!-- #9E9E9E -->
+
+      <md-icon layout="column" aria-label="" ng-style="{color: '#4CAF50'}" class="material-icons">lens</md-icon>
```

Comment:
Leaving as is, don't want to focus on thinking about CSS schema now, also as i said this approach is used all over Angular Material examples. Moved video's width style to scss.